components/motor/dimensionengineering/sabertooth_test.go

```diff
@@ -1,15 +1,16 @@
-package dimensionengineering_test
+package dimensionengineering
```

Comment:
I changed this package to easier test goForMath because a lot of other motor packages don't use this `_test` convention, but I am open to changing it back if there's a good reason to do so. It's not fully clear to me what this is useful for.

Reply:
This is actually a convention for making sure your test file is in a separate package which forces you to write good tests of the functionality of your exported APIs from that package. If you don't need any of the private functions in the dimensionengineering package to be tested now, please change it back. 

https://stackoverflow.com/questions/19998250/proper-package-naming-for-testing-with-the-go-language/31443271#31443271

Reply:
Sure I can revert this and remove the goForMath tests if you'd prefer

Reply:
> If you don't need any of the private functions in the dimensionengineering package to be tested now, please change it back.

You are testing private function now, so don't change it back.

Reply:
sorry I missed this before changing it, but I added GoFor tests so I think it should be fine now

Reply:
Even better!